juniper_hyper/Cargo.toml

```diff
@@ -14,11 +14,11 @@ url = "2"
 juniper = { version = "0.14.2", default-features = false, path = "../juniper"}
 tokio = "0.2"
 hyper = "0.13"
-futures = { version = "0.3.1" }
+futures = { version = "0.3.5" }
 
 [dev-dependencies]
 pretty_env_logger = "0.2"
-reqwest = "0.9"
+reqwest = { version = "0.10.4", features = [ "rustls-tls", "blocking" ] }
```

Comment:
And this one is good, but I'd rather suggest to use `0.10` as far as this is a dev dependency only, so we anyway will have the latest version executed for tests.